src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Automation/AbstractExtenderCATIDProvider.cs

```diff
@@ -21,41 +21,45 @@ internal abstract class AbstractExtenderCATIDProvider : IExtenderCATIDProvider
             // The latter issue we can do nothing about, however, to address the former, map these types to a truer form it makes 
             // it easier on implementors and maintainers of this to understand the objects we're talking about.
 
-            switch (extenderCATIDType)
+            return extenderCATIDType switch
             {
-                case ExtenderCATIDType.HierarchyExtensionObject:                                // IVsHierarchy.GetProperty(VSITEMID.Root, VSHPROPID_ExtObjectCATID)
-                case ExtenderCATIDType.AutomationProject:                                       // DTE.Project
-                    return GetExtenderCATID(ExtendeeObject.Project);
+                ExtenderCATIDType.HierarchyExtensionObject or                       // IVsHierarchy.GetProperty(VSITEMID.Root, VSHPROPID_ExtObjectCATID)
+                ExtenderCATIDType.AutomationProject =>                              // DTE.Project
```

Comment:
_This is not something you need to change for this PR, I just see an opportunity for a minor but interesting conversation._

When splitting an expression across multiple lines, I tend to put operators at the beginning of the next line rather than the end of the previous line. For example, instead of:

``` csharp
if (alpha ||
    beta)
{
    ...
}
```

I would write:

``` csharp
if (alpha
    || beta)
{
    ...
}
```

Of course, this example is so short that in practice I would put the whole expression on one line. 😀 But the advantage of this--when the expression has multiple subexpressions or is long--is that all the operators are lined up on the left and you can immediately see how all the subexpressions compose as you read, rather than waiting for your eye to reach the end of the line.

Reply:
For now, I won't change the style on this PR, but I agree that figuring out a standard style would be good. Given this example,
```C#
if (extension is
    ".fs" or
    ".fsi" or
    ".fsx" or
    ".fsscript" or
    ".ml" or
    ".mli")
```

Would we have to put the `is` on the next line? Like this?
```C#
if (extension
    is ".fs"
    or ".fsi"
    or ".fsx"
    or ".fsscript"
    or ".ml"
    or ".mli")
```

Or do you keep the `is` on the first line and just have the one, lonely value?
```C#
if (extension is
    ".fs"
    or ".fsi"
    or ".fsx"
    or ".fsscript"
    or ".ml"
    or ".mli")
```